curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8d539dc11cb42b5dd7ec8.md

```diff
@@ -7,7 +7,14 @@ dashedName: step-104
 
 # --description--
 
-The `shift()` method on an array removes the first element in the array and returns it. Use this method to take the first element from the `inventory` array and assign it to your `currentWeapon` variable.
+The `shift()` method on an array removes the first element in the array and returns it.
+
+```js
+const numbers = [1, 2, 3];
+const firstNumber = numbers.shift(); // returns 1
+```
+
+Use this method to take the first element from the `inventory` array and assign it to your `currentWeapon` variable.
```

Comment:
Let'e mention the method name again just like the issue mentioned 👍 
```suggestion
Use the `shift()` method to take the first element from the `inventory` array and assign it to your `currentWeapon` variable.
```

Reply:
Done :)